addons/web/static/src/legacy/frontend/rainbowman/rainbow_man.js

```diff
@@ -35,6 +36,7 @@ var RainbowMan = Widget.extend({
             fadeout: 'medium',
             img_url: '/web/static/img/smile.svg',
             message: _t('Well Done!'),
+            messageIsHtml: false,
```

Comment:
Just don't set the option.

Reply:
It's just a detail but I prefer defaulting it to `false` than keeping it undefined.